services/systemd/espeakup.service.in

```diff
@@ -6,10 +6,9 @@ After=systemd-udev-settle.service sound.target
 
 [Service]
 Type=forking
-EnvironmentFile=/etc/conf.d/espeakup
 PIDFile=/run/espeakup.pid
-ExecStartPre=+/sbin/modprobe speakup_soft
-ExecStart=/usr/bin/espeakup --default-voice=${default_voice}
+ExecStartPre=+modprobe speakup_soft
```

Comment:
So removing  `/sbin/` does work, did you test it?

Reply:
yep according to [systemd.service man](https://man.archlinux.org/man/systemd.service.5.en#COMMAND_LINES)

> If the command is not a full (absolute) path, it will be resolved to a full path using a fixed search path determined at compilation time. Searched directories include /usr/local/bin/, /usr/bin/, /bin/ on systems using split /usr/bin/ and /bin/ directories, and their sbin/ counterparts on systems using split bin/ and sbin/. It is thus safe to use just the executable name in case of executables located in any of the "standard" directories

it will work.